src/custom-apps/components/WebhookDetailsPage/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { gql } from "@apollo/client";
+
+export interface IntrospectionNode {
+  name: string;
+  interfaces: Array<{
+    name: string;
+  }> | null;
+  description: string;
+}
+
+// cannot be in `queries.ts` as codegen cannot handle `__schema`
+export const IntrospectionQuery = gql`
+  query EventsIntrospection {
+    __schema {
+      types {
+        name
+        interfaces {
+          name
+        }
+        description
+      }
+    }
+  }
+`;
+
+const isEvent = ({ name }) => name === "Event";
```

Comment:
suggestion: missing types for function params

Reply:
Done, utils  moved to `src/custom-apps/utils.tsx`